sdk/cosmos/azure-cosmos/README.md

```diff
@@ -681,10 +681,30 @@ indexing_policy = {
         ],
         "vectorIndexes": [
             {"path": "/vector1", "type": "flat"},
-            {"path": "/vector2", "type": "quantizedFlat"}
+            {"path": "/vector2", "type": "quantizedFlat"},
+            {"path": "/vector2", "type": "diskANN"}
         ]
     }
 ```
+
+For vector index types of diskANN and quantizedFlat, there are additional options available as well. These are:
+
+quantizationByteSize - the number of bytes used in product quantization of the vectors. A larger value may result in better recall for vector searches at the expense of latency. This applies to index types diskANN and quantizedFlat. The allowed range is between 1 and 256, and the default value is 64.
```

Comment:
Can we please make sure that guidance is same across, from JAVA

The allowed range for this parameter     * is between 1 and 3.
.NET also using the same as JAVA

Reply:
I wrote this down based on the exact error message that I received when testing against the account that had this enabled. Should we check with the query team on this to make sure then?

Reply:
Here is for .NET, I see similar for JAVA as well. Idea is to be consistent across. 

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/018dd2073b4f2697069de2b3e73af7cdae876f70/Microsoft.Azure.Cosmos/src/Resource/Settings/VectorIndexPath.cs#L72

/cc: @kundadebdatta